site/en/docs/extensions/reference/userscripts/index.md

```diff
@@ -0,0 +1,113 @@
+---
```

Comment:
Can we move this file to userScripts (capital S) in live with other APIs?

Reply:
Done.